03_launch_mgmt_cluster.sh

```diff
@@ -64,57 +64,87 @@ function clone_repos() {
 function patch_clusterctl(){
   pushd "${CAPM3PATH}"
   if [ -n "${CAPM3_LOCAL_IMAGE}" ]; then
-    CAPM3_IMAGE_NAME="${CAPM3_LOCAL_IMAGE##*/}"
-    export MANIFEST_IMG="${REGISTRY}/localimages/$CAPM3_IMAGE_NAME"
-    export MANIFEST_TAG="latest"
-    make set-manifest-image
+    CAPM3_IMAGE_NAME_WITH_TAG="${CAPM3_LOCAL_IMAGE##*/}"
+  else
+    CAPM3_IMAGE_NAME_WITH_TAG="${CAPM3_IMAGE##*/}"
+  fi
+  #  CAPM3_IMAGE include tag so split the CAPM3_IMAGE_NAME AND CAPM3_IMAGE_TAG
+  CAPM3_IMAGE_NAME="${CAPM3_IMAGE_NAME_WITH_TAG%%:*}"
+  CAPM3_IMAGE_TAG="${CAPM3_IMAGE_NAME_WITH_TAG##*:}"
```

Comment:
would this work if you have an image like "abc.com:5000/reg/image" or  "abc.com:5000/reg/image:latest"

Reply:
yes, it worked, I checked it locally

Reply:
You can check it from CI run in  BMO section, the image doesnot have any tag like your example.